File: HackMateBackend/HackMate/src/main/java/com/example/HackMateBackend/dtos/notification/NotificationListResponseDto.java
package com.example.HackMateBackend.dtos.notification;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class NotificationListResponseDto {
    private List<NotificationItemDto> notifications;
    private int unreadCount;
    private int totalCount;
    private boolean hasMore;
    private int page;
    private int size;
}

File: HackmateFrontendFolder/app/src/main/java/com/example/hackmatefrontendfolder/domain/model/notification/NotificationListResponseDto.kt
package com.example.hackmatefrontendfolder.domain.model.notification

data class NotificationListResponseDto(
    val notifications : List<NotificationItemDto>,
    val unreadCount : Int,
    val totalCount : Int,
    val hasMore : Boolean,
    val page : Int,
    val size : Int
)
